control-plane/connect-inject/common/common_test.go

```diff
@@ -314,3 +319,75 @@ func TestShouldIgnore(t *testing.T) {
 		})
 	}
 }
+
+func TestToProtoAny(t *testing.T) {
```

Comment:
Full disclosure, I didn't test panics in this function intentionally because 1) I'm not really sure how to trigger one easily and 2) the purpose of the function is use when doing things that should never panic.

Reply:
Let's keep it for now.

Reply:
Added handling for `anypb.Any` input -> return unmodified and test case to match.